<launch>

  <node pkg="test_detection" type="test_detection" name="test_detection" output="screen">

    <param name="map_vtk" value="$(find test_detection)/data/map/2.vtk"/>
    <param name="scan_vtk" value="$(find test_detection)/data/scan/7.vtk"/>
    <param name="pose" value="$(find test_detection)/data/pose/7.txt"/>

  </node>

<node pkg="rviz" type="rviz" name="rviz" args="-d $(find test_detection)/launch/test_detection.rviz" output="screen"/>
</launch>
